components/post_view/post/post.jsx

```diff
@@ -195,6 +196,7 @@ export default class Post extends React.PureComponent {
             (fromAutoResponder || !isSystemMessage) &&
             isEligibleForClick(e)
         ) {
+            trackEvent('crt', 'clicked_to_reply');
```

Comment:
this is going to have a ton of false positives!

Reply:
you mean people missclicking? or something else?

Reply:
sorry, I totally missed that, yep I meant people missclicking